dts/bindings/pinctrl/nuvoton,npcx-scfg.yaml

```diff
@@ -3,7 +3,7 @@
 
 description: Nuvoton, NPCX Pin-Controller (Pinmux, 1.8V support and so on) node
```

Comment:
We could change the description to state that this is a "System Configuration module"

Reply:
Done.